apix-impl/70/src/main/java/eu/xenit/apix/alfresco70/SearchServiceImpl70.java

```diff
@@ -0,0 +1,103 @@
+package eu.xenit.apix.alfresco70;
```

Comment:
specifically duplication could be lowered by extending the service once for functionality, then extend that class again to make it a non-abstract class per version:
```
abstract SearchService
-> Impl50
-> Impl51
-> abstract FunctionalExtension
---> Impl52
[...]
---> Impl70
```
We should have realised that when doing 52/60. I dont think its worth doing it now, since the 52 - 70 line is pretty consistent again.

Alternatively, we could have used interface extensions and/or default implementation (not a big fan of that last one though)